python/ray/air/util/data_batch_conversion.py

```diff
@@ -109,6 +109,58 @@ def convert_pandas_to_batch_type(
         )
 
 
+def _convert_batch_type_to_numpy(
+    data: DataBatchType,
+) -> Union[np.ndarray, Dict[str, np.ndarray]]:
```

Comment:
Can we add tests for these? See `test_data_batch_conversion.py`

Reply:
added !